packages/editor/src/components/provider/index.native.js

```diff
@@ -252,7 +252,7 @@ export default compose( [
 			title: getEditedPostAttribute( 'title' ),
 			getEditedPostContent,
 			selectedBlockIndex: getBlockIndex( selectedBlockClientId ),
-			blockCount: getBlockCount( rootClientId ),
+			blockCount: getGlobalBlockCount(),
```

Comment:
Looks like `getBlockCount` is not needed anymore and we can remove it from the extraction list above, right?